{"name": "CVE-2013-2132", "affected_release": {"advisory": "RHSA-2013:1170", "release_date": "2013-08-21T00:00:00", "cpe": "cpe:/a:redhat:enterprise_mrg:2:server:el6", "product_name": "MRG Grid for RHEL 6 Server v.2", "package": "pymongo-1.9-11.el6"}, "public_date": "2013-05-31T00:00:00", "package_state": [{"fix_state": "Will not fix", "product_name": "Red Hat CloudForms Tools 1", "package_name": "pymongo", "cpe": "cpe:/a:redhat:cloudforms_tools:1"}, {"fix_state": "Affected", "product_name": "Red Hat Satellite 6", "package_name": "pymongo", "cpe": "cpe:/a:redhat:network_satellite:6"}], "document_distribution": "Copyright \u00a9 2016 Red Hat, Inc. All rights reserved.", "threat_severity": "Moderate", "bugzilla": {"url": "https://bugzilla.redhat.com/show_bug.cgi?id=969560", "description": "\nCVE-2013-2132 pymongo: null pointer when decoding invalid DBRef\n    ", "id": "969560"}, "details": ["\nbson/_cbsonmodule.c in the mongo-python-driver (aka. pymongo) before 2.5.2, as used in MongoDB, allows context-dependent attackers to cause a denial of service (NULL pointer dereference and crash) via vectors related to decoding of an \"invalid DBRef.\"\n    "], "statement": "\nRed Hat Update Infrastructure 2.1.3 is now in Production 2 Phase of the support and maintenance life cycle. This has been rated as having Moderate security impact and is not currently planned to be addressed in future updates. For additional information, refer to the Red Hat Update Infrastructure Life Cycle: https://access.redhat.com/support/policy/updates/rhui.\n    ", "cwe": "CWE-476", "cvss": {"cvss_scoring_vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P", "status": "verified", "cvss_base_score": "5.0"}}